pandas/io/tests/test_packers.py

```diff
@@ -23,7 +24,12 @@
 
 nan = np.nan
 
-from pandas.io.packers import to_msgpack, read_msgpack
+try:
+    import blosc
+except ImportError:
+    _BLOSC_INSTALLED = False
```

Comment:
I know, but it doesn't actually *have* to build with python (e.g if you build it yourself).